lib/graphql/batch.rb

```diff
@@ -7,6 +7,7 @@
 module GraphQL
   module Batch
     BrokenPromiseError = ::Promise::BrokenError
+    class NoExecutorError < StandardError; end
```

Comment:
You have defined a second exception here without removing GraphQL::Batch::Loader::NoExecutorError.  You should instead make this an alias with `NoExecutorError = Loader::NoExecutorError` and use `deprecate_constant :NoExecutorError` in GraphQL::Batch::Loader.

Reply:
Ah good catch, done.